### PR TITLE
adding a production task, `npm run prod` that avoids browsersync and …

### PR DESCRIPTION
**Pull request by nf071590**
…watching files for changes. also added a timestamp to the browsersync prefix and updated the server task to a module that takes production flag as input.

### CODE REVIEW

**Comment by pertrai1** [comment]
@nf071590 Can you sign the new CLA that is on the repo: http://www.clahub.com/agreements/wevote/WebApp. This is part of the build process to check for a person that is doing a PR. Only have to agree to it once.

File: server.js [hunk 1]
```diff
@@ -2,15 +2,15 @@
 const express = require("express");
 const app = express();
 
-const port = 3003;
-const opts = {
-  redirect: true
-};
+module.exports = function (PROD) {
+  const port = PROD ? 3000 : 3003;
```
Comment (pertrai1) [inline server.js]: What is the reason behind the use of `const`?

Comment (nf071590) [inline server.js]: they are static variables that don't change.

Comment (pertrai1) [inline server.js]: cool

**Pull request merged**